themes/aframe/layout/partials/toc.ejs

```diff
@@ -0,0 +1,22 @@
+<% var toc = table_of_contents(page.content); %>
+
+<% if (toc.length) { %>
+  <h2>Table of Contents</h2>
+
+  <ul id="table-of-contents">
```

Comment:
also, fwiw, Wikipedia calls it just `toc`: https://en.wikipedia.org/wiki/Polar_bear#toc

want to change to that?